pkg/cli/improve.go

```
@@ -0,0 +1,250 @@
+	fmt.Fprintln(cmd.OutOrStdout(), "Kit improve")
+	fmt.Fprintln(cmd.OutOrStdout())
+	fmt.Fprintln(cmd.OutOrStdout(), "1. kit improve run --suite default")
+	fmt.Fprintln(cmd.OutOrStdout(), "2. kit improve mine --from .kit/improve/latest")
+	fmt.Fprintln(cmd.OutOrStdout(), "3. kit improve propose --from .kit/improve/latest")
+	fmt.Fprintln(cmd.OutOrStdout(), "4. kit improve validate --candidate <path>")
+	fmt.Fprintln(cmd.OutOrStdout(), "5. kit improve report --from .kit/improve/latest")
+	return nil
+}
+
+func currentGitCommit(projectRoot string) string {
+	out, err := exec.Command("git", "-C", projectRoot, "rev-parse", "HEAD").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)